### ISSUE TITLE
Examples here and  Twitter docs have very different approach

### ISSUE THREAD

**Issue by victor0x16** [twitterdev/Twitter-API-v2-sample-code#88]
Authorization flow here is very diffrent than the shown in the docs.

I'm just making a note here.

**Comment by andypiper** [comment]
Is this a reference to a specific example - could you point to the page in the docs, and the code sample you are looking at?

**Comment by victor0x16** [comment]
Yes, [this example here on github](https://github.com/twitterdev/Twitter-API-v2-sample-code/blob/main/Manage-Tweets/create_tweet.js) should be [like this in the docs](https://developer.twitter.com/en/docs/twitter-api/tweets/manage-tweets/api-reference/post-tweets). 

Also the authorization flow is quite different [in the docs](https://developer.twitter.com/en/docs/authentication/oauth-2-0) use a simple curl with oauth and [here](https://github.com/twitterdev/Twitter-API-v2-sample-code/blob/main/Manage-Tweets/create_tweet.js) use Pin Based Oauth but Pin Based oauth is not even mentioned in the docs.

Maybe it's just me not undertanding but I do think docs and here should sync. And look at [#this page](https://oauth-playground.glitch.me/?id=createTweet) is doing exactly what I want to do.

**Comment by andypiper** [comment]
Thanks for the feedback, we will discuss how to handle these thoughts - appreciate you taking the time to identify the areas we can improve!

**Comment by kennedybaird** [comment]
I'm also running into this problem - the code references CONSUMER_KEY and CONSUMER_SECRET

Are these supposed to be API_KEY and API_SECRET? That doesn't seem to work

**Comment by Imagineer99** [comment]
Did anyone get this to work? none of the sample codes seem to function.
